### PR TITLE
databases-week3

### PR DESCRIPTION
**Pull request by Hailegebriel22**

### CODE REVIEW

File: databases/week3/mealSharingDB.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
+CREATE DATABASE IF NOT EXISTS `Meal_sharing`;
+
+CREATE TABLE
```
Comment (Hanouj) [inline databases/week3/mealSharingDB.sql:3]: Great job here Haile you are really good at creating tables and connecting them together

File: databases/week3/mealSharingQueries.sql [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+        title,
+        description,
+        location,
+        `when`,
+        max_reservations,
+        price,
+        created
+    )
+values (
+        1,
```
Comment (Hanouj) [inline databases/week3/mealSharingQueries.sql]: Hi Haile nice work, but here id has an auto incremented primary key then there is an advantage that you do not need to give value for that column in the INSERT command. This means MySQL will give the value for that column

File: databases/week3/mealSharingQueries.sql [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+    number_of_guests,
+    meal_id,
+    created_date,
+    contact_phonenumber,
+    contact_name,
+    contact_email
+FROM Reservation
+    INNER JOIN meal ON Reservation.meal_id = meal.id
+WHERE meal.title = 'Buffet'
+ORDER BY created_date;
```
Comment (Hanouj) [inline databases/week3/mealSharingQueries.sql:322]: nice work Haile, here just add ORDER BY created_date ASC;
 to sort the result set in ascending order

**Review by Hanouj — COMMENTED** [review]
Great work on the homework Haile keep it up

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Haile really grear work. Optionally, you can  implement Hanouj's feedback regarding auto increment id(is a good catch) & then ship it.

File: databases/week3/mealSharingQueries.sql (hunk 3 above)
Comment (Hailegebriel22) [reply databases/week3/mealSharingQueries.sql:322]: Thanks, but the default is ASC.

**Review by Elia-Git — APPROVED** [review]
Ship it pls. Thanks!

**Pull request merged**